runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentification.java

```diff
@@ -239,10 +285,12 @@ enum ItemIdentification
 	BRONZE_BAR(Type.BAR, "Bronze", "BRO", ItemID.BRONZE_BAR),
 	IRON_BAR(Type.BAR, "Iron", "IRO", ItemID.IRON_BAR),
 	SILVER_BAR(Type.BAR, "Silver", "SIL", ItemID.SILVER_BAR),
+	LEAD_BAR(Type.BAR, "Lead", "LEA", ItemID.LEAD_BAR),
 	STEEL_BAR(Type.BAR, "Steel", "STE", ItemID.STEEL_BAR),
 	GOLD_BAR(Type.BAR, "Gold", "GOL", ItemID.GOLD_BAR),
 	MITHRIL_BAR(Type.BAR, "Mithril", "MIT", ItemID.MITHRIL_BAR),
 	ADAMANTITE_BAR(Type.BAR, "Adaman", "ADA", ItemID.ADAMANTITE_BAR),
+	CUPRONICKEL_BAR(Type.BAR, "Cupro.", "CUP", ItemID.CUPRONICKEL_BAR),
```

Comment:
Including a period in the medium name is only expected for separating words' abbreviations. Since that isn't happening here, it should be removed.
```suggestion
	CUPRONICKEL_BAR(Type.BAR, "Cupro", "CUP", ItemID.CUPRONICKEL_BAR),
```

Reply:
I had thought it was for all abbreviations but clearly that doesn't make sense or everything would have a period after it haha. Thanks for the spot, fixed!